src/components/transactions/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Waypoint from 'react-waypoint';
+import tableStyle from 'react-toolbox/lib/table/theme.css';
+import { transactions } from '../../utils/api/account';
+import TransactionsHeader from './transactionsHeader';
+import TransactionRow from './transactionRow';
+
+class Transactions extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      transactions: [],
+      offset: 0,
+      loadMore: true,
+      length: 1,
+    };
+  }
+
+  componentDidMount() {
+    this.loadMore();
+  }
+
+  loadMore() {
+    if (this.state.loadMore && this.state.length > this.state.offset) {
+      this.setState({ loadMore: false });
+      transactions(this.props.activePeer, this.props.address, 20, this.state.offset)
+      .then((res) => {
+        const list = res.transactions.map(transaction => (
+          <TransactionRow address={this.props.address}
+            key={transaction.id}
+            tableStyle={tableStyle}
+            value={transaction}>
+          </TransactionRow>
+        ));
+        this.setState({
+          transactions: this.state.transactions.concat(list),
```

Comment:
I think that the reason to dispatch `updateTransactions` action is that there will be also a reducer that removes them from pending transactions. I agree with handling 'pending transactions' in a separate issue

Also when I switch from and back to "transactions" tab it can immediately show the list from redux and then check for any updates.